src/constants/externalPages.ts

```diff
@@ -1,10 +1,10 @@
 export enum ExternalPages {
   LANDING = 'https://fluxity.finance',
-  DOCUMENTATION = '/documentation',
-  COMMUNITY = '/community',
   WHITEPAPER = '/white-paper',
+  DOCUMENTATION = '/documentation',
+  COMMUNITY = 'https://discord.gg/JnT7KPEuQs',
+  FLUXITY_API = 'https://api.fluxity.finance',
   EXPLORER = 'https://futurenet.stellarchain.io/accounts',
   FRIENDBOT = 'https://friendbot-futurenet.stellar.org/?addr=',
-  FLUXITY_API = 'https://api.fluxity.finance',
   EXPLORER_TRANSACTION = 'https://futurenet.stellarchain.io/transactions',
```

Comment:
we only need 1 field called EXPORER, without `accounts` or `transactions` inside the route.

Reply:
fix it whenever the EXPLORER_TRANSACTION and EXPLORER are used.

such that:

`${EXPLORER}/transactions/${hash}`

or

`${EXPLORER}/accounts/${address}`